R/constructNWISURL.r

```diff
@@ -335,7 +335,7 @@ constructWQPURL <- function(siteNumber,parameterCd,startDate,endDate,zip=FALSE){
 constructUseURL <- function(years,stateCd,countyCd,categories){ 
 
     if(is.null(stateCd)){
-      baseURL <- drURL("useNat", format="rdb", rdb_compression="value")
+      baseURL <- drURL("useNat", format="rdb", rdb_compression="value", Access=pkg.env$access)
```

Comment:
In the past, adding `Access` to some of the waterdata calls actually broke the links...that's why I was being deliberate. I'll run though all the NWIS services and see if that works now